arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "category": "Edit data",
-    "description": "Edit feature attributes which are linked to annotation through an expression.",
+    "description": "Edit feature attributes which are linked to annotations through an expression.",
```

Comment:
If you're going to make this change, please also update the description in content plist.

Reply:
Ahh really great catch! So which way should we go? 
In README it is "annotations" while in metadata and content plist it is "annotation".

Reply:
@vquach2404 Good catch. I just assumed this was fixing a discrepancy between the two.

Reply:
@yo1995 let's just change the metadata and plist to "annotations." I originally adjusted the README to use "annotations" instead.